sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -93,9 +94,11 @@ export interface EventDataAdapterParameters {
 
 // @public
 export interface EventDataBatch {
+    // @internal
+    _commitPublish(): void;
```

Comment:
if this is supposed to be hidden from the public surface, then I think it should be tagged as `@hidden` instead.

Reply:
I ended up removing them from the public surface.